pandas/core/frame.py

```diff
@@ -3772,6 +3772,11 @@ def lookup(self, row_labels, col_labels) -> np.ndarray:
         Given equal-length arrays of row and column labels, return an
         array of the values corresponding to each (row, col) pair.
 
+        .. deprecated:: 1.1.0
+
+        DataFrame.lookup is deprecated,
+        use DataFrame.melt and DataFrame.loc instead.
```

Comment:
how about you add an example in the docs of exactly this.

Reply:
and then point to it from here

Reply:
Where in the docs would I put the example?

Reply:
This docstring has some pre-existing issues so might need a few changes, but at the very least the first sentence should be `Label-based "fancy indexing" ...` to resolve the current CI complaint